src/main/kotlin/com/dotronglong/faker/service/plugin/CommandPlugin.kt

```diff
@@ -28,12 +28,10 @@ class CommandPlugin : Plugin {
                 return@create
             }
             try {
-                GlobalScope.launch {
-                    val commandPluginArguments = getCommandPluginArguments(arguments)
-                    val content = execute(commandPluginArguments)
-                    replace(response, commandPluginArguments, content)
-                    s.success()
-                }
+                val commandPluginArguments = getCommandPluginArguments(arguments)
```

Comment:
hi @snicoll . May I ask why should we remove GlobalScope in this case? Is there any rationale behide?

Reply:
Hi, I worked with @snicoll on those changes. If I remember correctly, there is no suspending functions invoked here so I thought it could be safely removed.